static/app/views/insights/common/components/insightsTimeSeriesWidget.tsx

```diff
@@ -128,16 +135,27 @@ export function InsightsTimeSeriesWidget(props: InsightsTimeSeriesWidgetProps) {
         alias: aliases?.[delayedTimeSeries.yAxis],
       });
     }),
-    ...(props.timeSeries?.filter(Boolean) ?? []).map(timeSeries => {
+    ...(props.timeSeries?.filter(Boolean) ?? []).map((timeSeries, idx) => {
       // TODO: After merge of ENG-5375 we don't need to run `markDelayedData` on output of `/events-timeseries/`
       const delayedTimeSeries = markDelayedData(timeSeries, INGESTION_DELAY);
 
       yAxes.add(timeSeries.yAxis);
 
+      let alias = aliases?.[delayedTimeSeries.yAxis];
```

Comment:
We can't always rely on the `yAxis` for `alias`. In the new `timeSeries` object, `yAxis` does not contain grouping information. To combat this I created `formatTimeSeriesName`, this is what the plottable now uses to generate it's `name` field, and we can also use that for the `alias` mapping.